store/CSDK/test/main.cpp

```diff
@@ -546,7 +546,7 @@ bool testWriteData(JNIEnv *env, char *path, int argc, char *argv[]) {
         writer.withCsvInput(jsonSchema);
         writer.withLoadOption("complex_delimiter_level_1", "#");
         writer.writtenBy("CSDK");
-        writer.taskNo(185);
+        writer.taskNo(15541554.81);
```

Comment:
This change not related to fix

Reply:
There are some different  between CPP and java: java taskNo don't support double, but CPP will convert double to long.